SDK/AppCenterData/Microsoft.AppCenter.Data.Shared/ConversionExtensions.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+
+namespace Microsoft.AppCenter.Data
+{
+    public static class SharedConversionExtensions
+    {
+        public static DocumentWrapper<T> ToDocumentWrapper<T>(
```

Comment:
nit: We don't usually format function signatures like this, just put them in line and then wrap when the line gets too long